src/components/portfolio/list/PortfolioItem.tsx

```diff
@@ -0,0 +1,166 @@
+import { useMediaQuery } from 'react-responsive';
+
+import styled from '@emotion/styled';
+import { ModalType, useModalStore } from 'stores/useModalStore';
+import { PortfolioData } from 'types/portfolio';
+
+import PortfolioItemSkills from './PortfolioItemSkills';
+
+const PortfolioItem = ({ no, title, skills, start, end }: PortfolioData) => {
+  const { setIsModalOn, modalState } = useModalStore((state) => ({
+    setIsModalOn: state.setIsModalOn,
+    modalState: state.modalState,
+  }));
+
+  const openModal = () => {
+    setIsModalOn(ModalType.PORTFOLIO_DETAILED, true);
+  };
+
+  const isSmallDisplay = useMediaQuery({ query: '(max-width: 320px)' });
+  const isMediumDisplay = useMediaQuery({ query: '(max-width: 768px)' });
+
+  if (isSmallDisplay) {
+    return (
+      <Item onClick={openModal}>
+        <SmallWrapper>
+          <CardNo>{String(no).padStart(2, '0')}</CardNo>
+          <TitleText>{title}</TitleText>
+          <InfoDate>
+            {start} - {end}
+          </InfoDate>
+        </SmallWrapper>
+      </Item>
+    );
+  }
+
+  if (isMediumDisplay) {
+    return (
+      <Item onClick={openModal}>
+        <MediumWrapper>
+          <CardNo>{String(no).padStart(2, '0')}</CardNo>
+          <MediumTitleWrapper>
+            <TitleText>{title}</TitleText>
+            <PortfolioItemSkills skills={skills} />
+          </MediumTitleWrapper>
+        </MediumWrapper>
+        <InfoDate>
+          {start} - {end}
+        </InfoDate>
+      </Item>
+    );
+  }
+
+  return (
+    <Item onClick={openModal}>
+      <TitleWrapper>
+        <CardNo>{String(no).padStart(2, '0')}</CardNo>
+        <TitleText>{title}</TitleText>
+      </TitleWrapper>
+      <InfoWrapper>
+        <InfoDate>
+          {start} - {end}
+        </InfoDate>
+        <PortfolioItemSkills skills={skills} />
+      </InfoWrapper>
+    </Item>
+  );
+};
+
+const Item = styled.li`
+  display: flex;
+  justify-content: space-between;
+  ${({ theme: { colors } }) => ({
+    color: colors.MAIN_FONT,
+  })};
+  border-bottom: 0.1rem solid ${({ theme }) => theme.colors.MAIN_FONT};
+  align-items: center;
+  padding: 4rem 0;
+  ${({ theme: { breakPoint } }) => {
+    return {
+      [breakPoint.medium]: {
+        alignItems: 'flex-start',
+        padding: '6rem 0',
+      },
+    };
+  }}
+`;
+
+const TitleWrapper = styled.div`
+  display: flex;
+  gap: 2rem;
+`;
+
+const CardNo = styled.p`
+  font-size: 3.2rem;
+  font-family: 'Bebas Neue', cursive;
```

Comment:
[SUG] cursive로 하면 메인 폰트 로드 전까지 이상한 필기체가 나와서 제가 이전에 수정한걸로 바꿔주시면 좋겠습니다.

```suggestion
  font-family: 'Bebas Neue', -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Oxygen, Ubuntu,
    Cantarell, Fira Sans, Droid Sans, Helvetica Neue, sans-serif;```
```